src/absil/ilwritepdb.fs

```diff
@@ -125,6 +126,30 @@ type idd =
       iddData: byte[]
       iddChunk: BinaryChunk }
 
+/// The specified Hash algorithm to use on portable pdb files.
+type HashAlgorithm =
+| Sha1
+| Sha256
```

Comment:
Same

Reply:
Sigh!!! I prefer aligning enums under the type.